### PR TITLE
Avoid non portable == shell test usage

### PR DESCRIPTION
**Pull request by voidpin**
Please avoid the use of non portable "==" shell test usage.
Thanks!

I have to check how to avoid this also,
 ```
36 if(${CMAKE_SYSTEM_NAME} MATCHES "OpenBSD" OR ${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
 37     message(WARNING "${CMAKE_SYSTEM_NAME} is not supported by lxqt-admin-user")
 38     message(WARNING "${CMAKE_SYSTEM_NAME} is not supported by lxqt-admin-time")
 39 else()
 40     add_subdirectory(lxqt-admin-user)
 41     add_subdirectory(lxqt-admin-time)
 42 endif() 
```  
This is unnecessary. `lxqt-admin-user` runs perfectly fine on NetBSD. Only `lxqt-admin-time` needs `systemd`. For packaging on NetBSD, I have a work around but currently it basically just excludes `lxqt-admin-time` from the package.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -34,8 +34,7 @@ include(LXQtCompilerSettings NO_POLICY_SCOPE)
 include(LXQtTranslate)
 
 if(${CMAKE_SYSTEM_NAME} MATCHES "OpenBSD" OR ${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
-    message(WARNING "${CMAKE_SYSTEM_NAME} is not supported by lxqt-admin-user")
-    message(WARNING "${CMAKE_SYSTEM_NAME} is not supported by lxqt-admin-time")
+    add_subdirectory(lxqt-admin-user)
```
Comment (palinek) [inline CMakeLists.txt]: What's the solution?

Comment (voidpin) [reply CMakeLists.txt]: Wouldn't this work?
```
if(${CMAKE_SYSTEM_NAME} MATCHES "OpenBSD" OR ${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
    add_subdirectory(lxqt-admin-user)
else()
    add_subdirectory(lxqt-admin-user)
    add_subdirectory(lxqt-admin-time)
endif()

```
lxqt-admin-user works just fine. Its only lxqt-admin-time that causes problems due to systemd dependency.

Comment (palinek) [reply CMakeLists.txt]: > lxqt-admin-user works just fine. Its only lxqt-admin-time that causes problems due to systemd dependency.

Well then
1. the commit message is misleading; please, change it to not make confusion
2. please, leave the warning about not supported `lxqt-admin-time`

**Comment by voidpin** [comment]
Yes, I understand. When commit it went through with the first pull request.
As said give a couple of hours to build and I'll report back.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by palinek** [comment]
> As said give a couple of hours to build and I'll report back.

Take your time...

**Comment by palinek** [comment]
Please, squash commits 8842dd6 and cf7855b

**Comment by voidpin** [comment]
Maybe easier to close these and create new pull requests?!

lxqt-sudo DONE.
lxqt-admin will start build soon.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by palinek** [comment]
> Maybe easier to close these and create new pull requests?!

You are free to choose...

**Comment by voidpin** [comment]
I'm closing this and the one for lxqt-sudo and submit new ones. The packages build fine after the changes.

**Pull request closed without merging**